helm/sealed-secrets/templates/ingress.yaml

```diff
@@ -12,12 +12,16 @@ metadata:
     {{- end }}
   {{- end }}
 spec:
+  {{- if .Values.ingress.tls }}
+  tls:
+  {{ tpl (toYaml .Values.ingress.tls | indent 4) . }}
+  {{- end }}
```

Comment:
The `.Values.ingress.tls` isn't a map anymore but a boolean, see:
- https://github.com/bitnami-labs/sealed-secrets/blob/main/helm/sealed-secrets/values.yaml#L164
You can use `.Values.ingress.extraTls` to add TLS configuration additional hostname(s), see:
- https://github.com/bitnami-labs/sealed-secrets/blob/main/helm/sealed-secrets/values.yaml#L192